kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -329,7 +329,7 @@ def _get_consumer_offsets(self):
         else:
             raise ConfigurationError(
                 "Cannot fetch consumer offsets because no consumer_groups are specified and "
-                "monitor_unlisted_consumer_groups is %s." % self.instance._monitor_unlisted_consumer_groups
+                "monitor_unlisted_consumer_groups is %s." % self._monitor_unlisted_consumer_groups
             )
```

Comment:
I guess this code path is not covered by tests. Do you mind adding a test for this case ?